.gitlab/rzvernal-jobs.yml

```diff
@@ -0,0 +1,28 @@
+#############################################################################
+## Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+## and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+##
+## SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+# HIP jobs on RZvernal
+
+#hip_5_1_0_cce_13_0_0:
```

Comment:
Yes. So in order to use a CCE compiler, we need v14.0.0 in order to use rocm@5.1.0... I have not tried with like rocm@4.5.2 and CCE@13.0.2 though... I found out that CCE@13.0.0 does not work though - with either rocm 5.1.0 or 4.5.2

Reply:
(And CCE@14.0.0 is still in the works... afaik)

Reply:
Right. I remember the story now. Thanks for keeping up with it.

Reply:
I just pulled in the radiuss updates for this thru RAJA so that I can use the updates in RAJAPerf. Once that's merged in, the rznevada tests will start passing again.. Still have to figure out some rzvernal + Github CI issues before that pipeline works, but the local build with the configuration I have will work.